### ISSUE TITLE
Docker Image not formatted for Raspberry Pi (linux/arm64/v8)

### ISSUE THREAD

**Issue by brettdavies** [indiefan/home_assistant_nanit#1]
@indiefan thank you for improving the no-longer-maintained project.

Currently, the docker image does not work on my Raspberry Pi 4 running Raspbian 11 Bullseye.

Can we add an image for the Raspberry Pi (e.g., linux/arm64/v8) platform in addition to the existing desktop(?) platform (e.g., linux/amd64)?

It should be straightforward to add a second supported platform as part of the current build process. I am happy to help if you need assistance.

```
user@raspberry:~ $ docker run -it -v /nanit/data:/data --entrypoint=/app/scripts/init-nanit.sh indiefan/nanit
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
standard_init_linux.go:219: exec user process caused: exec format error
```

**Comment by indiefan** [comment]
Thanks for pointing this out.

I haven't done this before, but I can look into it tomorrow. That said, if you know what needs to be done already and want to send a pull request, I'd happily accept it.

**Comment by indiefan** [comment]
Okay, I took a crack at this. Try it out and let me know if I did it wrong! Thanks again for bringing it to my attention.

**Comment by brettdavies** [comment]
Yes, that is the most straightforward way to do it. I am glad you are already using Github actions. I've pulled the indiefan/nanit:latest image and can confirm it opens fine on my Pi.

It is OK to close this issue.
